.github/workflows/ci.yaml

```
@@ -79,6 +79,19 @@ jobs:
         run: nox -R -e "$NOX_SESSION"
         timeout-minutes: 10
 
+  # This job runs if all the `nox` matrix jobs ran and succeeded.
+  # It is only used have a single job we can require in branch protection
```

I think this sentence needs to be rephrased. Maybe something around
```suggestion
  # It is only used to define a single job that we can require in branch protection
```

---

Thanks, updated (I used "It is only used to have ...")